docs/_includes/footer.html

```diff
@@ -0,0 +1,28 @@
+<footer class="site-footer h-card">
+
+  <div class="footer-info-box">
+    <ul class="stipulation">
+      <li>이용약관</li>
+      <li>개인정보처리방침</li>
+      <li>전자금융거래이용약관</li>
+    </ul>
+
+    <div class="company-info">
+      <p>경기도 성남시 분당구 대왕판교로 644번길 49 한컴타워</p>
+      <p>사업자등록번호 107-81-52230   ㅣ   통신판매신고번호 2012-경기성남-1092   ㅣ   (주)한글과컴퓨터 대표자 변성준</p>
+    </div>
+
+    <div class="copyright">
+      Copyrights ©Hancom Inc. All rights reserved.
```

Comment:
```suggestion
      Copyrights © Hancom Inc. All rights reserved.
```
© 뒤에 공백 들어가야 하는 건 아닌가요?

Reply:
아 그렇습니까? 다음 번 커밋에 반영하도록 하겠습니다~

Reply:
Copyrights와 중복되니 "©" 제거해주세요.(예전회의 언급)